infra/helper.py

```diff
@@ -85,6 +85,8 @@
 CLUSTERFUZZLITE_FILESTORE_DIR = 'filestore'
 CLUSTERFUZZLITE_DOCKER_IMAGE = 'gcr.io/oss-fuzz-base/cifuzz-run-fuzzers'
 
+logger = logging.getLogger(__name__)
```

Comment:
Would it make more sense to put "helper" here instead?

Reply:
This is the idiomatic way of doing this. When imported as "import helper", __name__ automatically becomes "helper".